Reference/Routing/WebApi/authorization.md

```diff
@@ -1,18 +1,22 @@
 #Umbraco Api - Authorization
 
-_This section will describe how to secure your Umbraco Api controllers based on a users membership_ 
+_This section will describe how to secure your Umbraco Api controllers based on a users membership._ 
 
 ##Authorizing for the back office
 
 ###Inheriting from UmbracoAuthorizedApiController
 
-Probably the easiest way to ensure your controller is secured for only back office users is to inherit from `Umbraco.Web.WebApi.UmbracoAuthorizedApiController`. This is essentially the same as applying the `UmbracoAuthorizeAttribute` to your controller (see below). We also expose the Umbraco user property `User UmbracoUser {get;}` on this base controller as well.
+Probably the easiest way to ensure your controller is secured for only back office users is to inherit from `Umbraco.Web.WebApi.UmbracoAuthorizedApiController`. This is similar to applying the `IsBackOfficeAttribute` to your controller (see below), but it will ensure all actions will be secured. We also expose the Umbraco user property `User UmbracoUser {get;}` on this base controller as well.
 
-The `UmbracoAuthorizedApiController` is automatically routed.  Check out the [routing documentation](../Authorized/index.md) for more information on this topic.
+Controllers inheriting from `UmbracoAuthorizedApiController`, or ones that have the `IsBackOfficeAttribute` are automatically routed. Check out the [routing documentation](../Authorized/index.md) for more information on this topic.
 
 ###Using UmbracoAuthorizeAttribute
 
-To secure your controller based on back offce membership use the attribute: `Umbraco.Web.WebApi.UmbracoAuthorizeAttribute`. 
+Using this attribute in conjunction with `IsBackOfficeAttribute` will allow you to secure some actions but still leave others open. If you don't use `IsBackOfficeAttribute` on your controller, then `UmbracoAuthorizeAttribute` will always cause your actions to return unauthorised, regardless of if you are logged into the back office.
```

Comment:
Again i think there is some confusion as to what IsBackOfficeAttribute is doing so this isn't correct. The `IsBackOfficeAttribute` is for one thing - auto-routed UmbracoApiControllers with this attribute are routed as back office routes so they can be authenticated as back office requests.

If you use `UmbracoAuthorizeAttribute` then it will do an authorization check to make sure that there is a back office user logged in, otherwise the resource cannot be accessed. This can be applied at the controller level which will secure all actions on the controller, or at the action level which can secure individual actions.